src/components/Tutorial/NavigationBar.vue

```diff
@@ -21,7 +21,7 @@
           :to="urlWithParams"
         >
           <template slot="default">{{ technology }}</template>
-          <template slot="subhead">Tutorials</template>
+          <template slot="subhead">{{ $tc('tutorials.title', 2) }}</template>
```

Comment:
```suggestion
          <template slot="subhead">{{ $tc('tutorials.title', 1) }}</template>
```

Reply:
I think `2` is more correct since we want the plural "Tutorials" here and not the singular "Tutorial".

Reply:
I wanted to ask if it's 0 indexed or 1 indexed, because from other places it seemed like it was 0 indexed?

Reply:
Yes, we want it to be plural so we should use `2` instead of `1`.
In case that it could be `0` then I would need to add a third string in the translation file. You can find more info here:
https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument
They explain it better than I do 😄